srcpkgs/gnuradio-nrsc5/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'gnuradio-nrsc5'
+pkgname=gnuradio-nrsc5
+version=20180917
+revision=1
+_githash=e3f06d9acc0f3cc915b2296c717476ad5b045b05
```

Comment:
please ask upstream to tag a release~

Reply:
asked https://github.com/argilo/gr-nrsc5/issues/9.